src/HealthChecks.OpenIdConnectServer/OidcConstants.cs

```diff
@@ -0,0 +1,22 @@
+namespace HealthChecks.IdSvr;
+
+internal class OidcConstants
+{
+    internal const string ISSUER = "issuer";
+
+    internal const string AUTHORIZATION_ENDPOINT = "authorization_endpoint";
+
+    internal const string JWKS_URI = "jwks_uri";
+
+    internal const string RESPONSE_TYPES_SUPPORTED = "response_types_supported";
+
+    internal const string SUBJECT_TYPES_SUPPORTED = "subject_types_supported";
+
+    internal const string ALGORITHMS_SUPPORTED = "id_token_signing_alg_values_supported";
+
+    internal static IEnumerable<string> REQUIRED_RESPONSE_TYPES => new[] { "code", "id_token", "token id_token" };
+
+    internal static IEnumerable<string> REQUIRED_SUBJECT_TYPES => new[] { "pairwise", "public" };
+
+    internal static IEnumerable<string> REQUIRED_ALGORITHMS => new[] { "RS256" };
```

Comment:
```suggestion
    internal static string[] REQUIRED_RESPONSE_TYPES => new[] { "code", "id_token", "token id_token" };

    internal static string[] REQUIRED_SUBJECT_TYPES => new[] { "pairwise", "public" };

    internal static string[] REQUIRED_ALGORITHMS => new[] { "RS256" };
```

Reply:
Same question as above. Here I am using only `Contains(...)` method  so I thought I would use the most most basic collection interface.

Reply:
>  so I thought I would use the most most basic collection interface.

Indeed, but it's not needed when working in internal (i.e. 100% your) code.